LanguageFeatures/primary-constructors/static_processing_A09_t03.dart

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) 2025, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion The following errors apply to formal parameters of a declaring
+/// constructor, be it in the header or in the body. Let `p` be a formal
+/// parameter of a declaring constructor in a class, enum, or extension type
+/// declaration `D` named `C`:
+/// ...
+/// A compile-time error occurs if `p` has the modifier `covariant`, but neither
+/// `var` nor `final`.
+///
+/// @description Check that it is a compile-time error if a formal parameters of
+/// a declaring constructor has the modifier `covariant`, but neither `var` nor
+/// `final`. Test enums.
```

Comment:
Having `covariant` but neither `final` nor `var` is an error, and not having `final` is another error in the first place, so it could be claimed that we don't know what we're testing (and a bug could be undetected because we think we're testing one of them, but it's the other that causes the error).

However, I think it's useful to see that the lone `covariant` is handled (and not crashing, for example). So let's keep it.